pkg/tilesets/tilesets.go

```diff
@@ -141,3 +141,134 @@ func StandardTileSet() TileSet { //nolint:gocyclo // shallow loops for adding ti
 		Tiles:        tiles,
 	}
 }
+
+func MiniTileSet() TileSet {
+	var tiles = []tiles.Tile{}
+	// mini simple set containing (12 tiles in total):
+	// 1 monastery with road
+	// 2 straight roads
+	// 1 straight road with city
+	// 3 road turns
+	// 2 T crossroads
+	// 3 city edges up and down not connected
+
+	// 1 monastery with road
+	tiles = append(tiles, tiletemplates.MonasteryWithSingleRoad())
+
+	// 2 straight roads
+	tiles = append(tiles, tiletemplates.StraightRoads())
+	tiles = append(tiles, tiletemplates.StraightRoads())
+
+	// 1 straight road with city
+	tiles = append(tiles, tiletemplates.SingleCityEdgeStraightRoads())
+
+	// 3 road turns
+	tiles = append(tiles, tiletemplates.RoadsTurn())
+	tiles = append(tiles, tiletemplates.RoadsTurn())
+	tiles = append(tiles, tiletemplates.RoadsTurn())
+
+	// 2 T crossroads
+	tiles = append(tiles, tiletemplates.TCrossRoad())
+	tiles = append(tiles, tiletemplates.TCrossRoad())
+
+	// 3 city edges up and down not connected
+	tiles = append(tiles, tiletemplates.TwoCityEdgesUpAndDownNotConnected())
+	tiles = append(tiles, tiletemplates.TwoCityEdgesUpAndDownNotConnected())
+	tiles = append(tiles, tiletemplates.TwoCityEdgesUpAndDownNotConnected())
+	return TileSet{
+		StartingTile: tiletemplates.SingleCityEdgeStraightRoads(),
+		Tiles:        tiles,
+	}
+}
+
+func OrderedMiniTileSet1() TileSet { //nolint:gocyclo
+	var tiles []tiles.Tile
+	// mini simple set containing (12 tiles in total):
+	// 1 monastery with road
+	// 2 straight roads
+	// 1 straight road with city
+	// 3 road turns
+	// 2 T crossroads
+	// 3 city edges up and down not connected
+
+	tiles = append(tiles, tiletemplates.SingleCityEdgeStraightRoads().Rotate(2)) // turn 1
+	tiles = append(tiles, tiletemplates.RoadsTurn())
+	tiles = append(tiles, tiletemplates.RoadsTurn().Rotate(1)) // turn 3
+	tiles = append(tiles, tiletemplates.TCrossRoad().Rotate(3))
+	tiles = append(tiles, tiletemplates.MonasteryWithSingleRoad().Rotate(2)) // turn 5
+	tiles = append(tiles, tiletemplates.TwoCityEdgesUpAndDownNotConnected().Rotate(1))
+	tiles = append(tiles, tiletemplates.TwoCityEdgesUpAndDownNotConnected().Rotate(1)) // turn 7
+	tiles = append(tiles, tiletemplates.StraightRoads().Rotate(1))
+	tiles = append(tiles, tiletemplates.TCrossRoad().Rotate(3)) // turn 9
+	tiles = append(tiles, tiletemplates.TwoCityEdgesUpAndDownNotConnected().Rotate(1))
+	tiles = append(tiles, tiletemplates.RoadsTurn().Rotate(2)) // turn 11
+	tiles = append(tiles, tiletemplates.StraightRoads())
+
+	return TileSet{
+		StartingTile: tiletemplates.SingleCityEdgeStraightRoads(),
+		Tiles:        tiles,
+	}
+}
+
+func OrderedMiniTileSet2() TileSet { //nolint:gocyclo
```

Comment:
The "Ordered" tilesets are not any more ordered than any other tilesets. Maybe some other name would be more appropriate?

Reply:
can't come with better name